projects/fluid_simulation/README.md

```diff
@@ -8,106 +8,4 @@ from the original WebGL implementation in https://github.com/PavelDoGreat/WebGL-
-dissipation and smoother density fields.
-
-## `--dye-resolution` (float, default: 1024)
-This determines the level of detail in which the dye is represented.  This
-changes the clarity of the dye patterns in the simulation.  Higher values
-provide finer details and sharper patterns.
-
-## `--enable-debug` (bool, default: false)
-If given, this enables additional debugging output.  Mainly, it shows the textures used during
-computation.
-
-## `--enable-marble` (bool, default: true)
-When set, this instantiates a marble that bounces around the simulation field. The marble bounces
-above the fluid, but it splashes down with certain frequency (controlled by `--marble-drop-frequency`).
-**NOTE**: This option is not available in the original WebGL implementation.
-
-## `--marble-drop-frequency` (float, default: 0.1)
-The probability that the marble will splash on the fluid as it bounces around the field.
-
-## `--num-splats` (int, default: 0)
-This is the number of splashes of color to use at the start of the simulation.
-This is also used when `--splat-frequency` is given (see below). A value of `0`
-means "random number of splats".
-
-## `--pressure` (float, default: 0.8)
-Indicates the force exerted by the fluid on its surrounding boundaries. Higher values cause
-a greater force exerted on the boundaries. This can lead to denser regions in the fluid.
-
-## `--pressure-iterations` (int, default: 20)
-This is the number of iterations performed when solving the pressure field.
-Higher values produce a more accurate and detailed pressure computation.
-
-## `--sim-resolution` (int, default: 128)
-This determines the grid size of the compute textures used during simulation.
-Higher values produce finer grids which produce a more accurate representation.
-
-## `--splat-force` (float, default: 6000.0)
-This represents the magnitude of the impact applied when an external force (e.g. marble drops) on
-the fluid.
-
-## `--splat-frequency` (float, default: 0.1);
-How frequent should new splats be generated at random.  The number of splats is
-controlled by `--num-splats`.
-
-## `--splat-radius` (float, default: 0.25)
-This represents the extent of the influence region around a specific point
-where the splat force is applied.
-
-## `--enable-sunrays` (bool, default: true)
-This enables the effect of rays of light shining through the fluid.
-
-## `--sunrays-resolution` (int, default: 196)
-Indicates the level of detail for the light rays. Higher values produce a finer
-level of detail for the light.
-
-## `--sunrays-weight` (float, default: 1.0)
-Indicates the intensity of the light scattering effect.  Higher values result
-in more prominent sun rays, making them appear brighter.
-
-## `--velocity-dissipation` (float, default: 0.2)
-This simulates the loss of energy within the fluid system. Higher values result
-in faster velocity reduction.
+Check `--help` to see more details on flag usage.
```

Comment:
please leave these here.  Documentation should not require building the binary.  Every time the code changes, this should be changed.  Yes, it creates a double repository for documentation but this is easy to re-create by running the binary with `--help` whenever flags change.